src/odemis/gui/util/test/updater_test.py

```diff
@@ -39,7 +39,7 @@ def test_version(self):
         rv = u.get_remote_version()
         # Note: if no internet access, it would be acceptable to get None
 
-        self.assertIn(len(rv.split('.')), (3, 4))
+        self.assertIn(len(rv.split(b'.')), (3, 4))
```

Comment:
I think here the test case is correct, but `get_remote_version()` should return a (unicode) string.